views/index.pug

```diff
@@ -4,6 +4,7 @@ html
     script(defer src="https://use.fontawesome.com/releases/v5.0.9/js/all.js" integrity="sha384-8iPTk2s/jMVj81dnzb/iFR2sdA7u06vHJyyLlAd4snFpCl/SnyUjRrbdJsw1pGIl" crossorigin="anonymous")
     script(src='http://d3js.org/d3.v2.min.js?2.9.3')
     link(rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous")
+    link(rel="manifest" href="https://raw.githubusercontent.com/andywong418/moneyd-gui/master/static/pay/manifest.json")
```

Comment:
if this is in our static files, can't we pull it from this server using a relative URL?